src/Uno.UI.Tests/Windows_UI_XAML_Controls/TextBlockTests/WhiteSpace_In_TextBlock_Inlines.xaml

```diff
@@ -8,8 +8,8 @@
     mc:Ignorable="d"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
 
-    <StackPanel>
-		<TextBlock x:FieldModifier="public" x:Name="tbDefault" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"><Bold>Word1</Bold> <Bold>Word2</Bold>  <Bold>Word3</Bold> LeftSpace<Run>Word4</Run>RightSpace <Run>Word5</Run> SurroundSpace <Run>Word6</Run>Middle Space<Run>Word7 </Run>  DoubleSpace  </TextBlock>
-		<TextBlock x:FieldModifier="public" x:Name="tbPreserve" xml:space="preserve" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"><Bold>Word1</Bold> <Bold>Word2</Bold>  <Bold>Word3</Bold> LeftSpace<Run>Word4</Run>RightSpace <Run>Word5</Run> SurroundSpace <Run>Word6</Run>Middle Space<Run>Word7 </Run>  DoubleSpace  </TextBlock>
+	<StackPanel>
+		<TextBlock x:FieldModifier="public" x:Name="tbDefault" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">Word0<Bold>Word1</Bold><Bold>Word2</Bold>  <Bold>Word3</Bold> LeftSpace<Run>Word4</Run>RightSpace <Run>Word5</Run> SurroundSpace <Run>Word6</Run>Middle Space<Run>Word7 </Run>  DoubleSpace</TextBlock>
+		<TextBlock x:FieldModifier="public" x:Name="tbPreserve" xml:space="preserve" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation">Word0<Bold>Word1</Bold><Bold>Word2</Bold>  <Bold>Word3</Bold> LeftSpace<Run>Word4</Run>RightSpace <Run>Word5</Run> SurroundSpace <Run>Word6</Run>Middle Space<Run>Word7 </Run>  DoubleSpace</TextBlock>
```

Comment:
Why are we changing test input?

Reply:
This is because the result is different depending on whether a text is placed in front or an element is placed. But I have to fix the xamlreader again because unit test is failed.

Reply:
I'm not following.

I think we should keep the test case as it is.

If we wanted to test something different, this should be done in a separate TextBlock (ie, add *new* test input rather than modifying *existing* test input)

Reply:
Ok, I will follow your opinion.

Reply:
I've done